### DISCUSSION TITLE
What's Next?

### DISCUSSION THREAD

**Discussion by rammyfzl** [Security-Onion-Solutions/securityonion#2655] [Unsupported Versions]
Hi there,

I am very new to SO and I've been able to get the SO v2.3.21 evaluation installation completed. So far I have added ubuntu and windows hosts to fleet and Wazuh (both are VMs on the same network as SO VM). But...what's next? I feel like I'm missing something. I launched a portscan and vuln scan against all of the hosts from within the network, but have not been able get any alerts just yet. I am accessing SOC from an Analyst VM as well as directly from SO.

**Troubleshooting Steps**
- I have confirmed that SO is seeing the network traffic; ran tcpdump while performing the scans
- Activated half of the rulesets in Playbook (though these seem to be Windows related)

**Findings**
- In Kibana I can only see 2 alerts from the offending VM, but these do not give much info if any
- In the Alerts tab I found a medium sev alert (Interface entered in promiscuous(sniffing) mode.) and low sev alert (sshd: insecure connection attempt (scan)) in relation to the activity from the offending host, but nothing noteworthy.

Is there something that I missed? Are there further guides/courses that I missed?
Many thanks

**Comment by weslambert** [comment]
With regard to the traffic, I would first try replaying some traffic that you expect to be alerted upon, based on the loaded NIDS rules, etc.  For example, for NIDS/Suricata, ensure that the rule is actually loaded in `/opt/so/rules/nides/all.rules`, then replay a PCAP which matches the content for which the rule is searching.

**Reply by rammyfzl** [reply]
Thank you for your response. I was able to capture and replay malicious traffic. I received several alerts including Zeek and Suricata. Whilst this is good, is there any way to capture the traffic real-time in the evaluation build?

Also, I see that the so-test issue has been fixed in  [#2463](https://github.com/Security-Onion-Solutions/securityonion/issues/2463). How can I apply the fix to my current build?

**Reply by weslambert** [reply]
Eval mode should allow you to monitor traffic in real-time, just as the other modes do.  I see you've also asked [here](https://github.com/Security-Onion-Solutions/securityonion/issues/2463#issuecomment-763199634) about how to apply the fix, etc, so I won't repeat the answer here.

**Reply by rammyfzl** [reply]
Thank you. Are there any troubleshooting steps for monitoring in real-time? I'd like to get that going before installing the standalone/distributed version.

**Reply by rammyfzl** [reply]
I've attached a dump of the suricata.yaml file if that helps. I also ensured that all the docker services are running and started the so-suricata service just in case. I then ran thru the same exercise captured in the PCAP that was replayed, but no dice. Also, unsure if this is related, but for the replayed activity, in Hunt, I don't get the option to "Show PCAP for this event" when expanding the alerts.

[cat :opt:so:conf:suricata:suricata_yaml.txt](https://github.com/Security-Onion-Solutions/securityonion/files/5863702/cat.opt.so.conf.suricata.suricata_yaml.txt)

**Reply by weslambert** [reply]
Also, to be clear, the Playbook rules only apply to searching log data that has already been recorded, not necessarily to NIDS alerts generation itself.  You'll want to check to make sure that the same rules you are looking to have triggered are enabled (not commented out) in `/opt/so/rules/nids/all.rules`.

**Reply by rammyfzl** [reply]
No these are Suricata rules and they are enabled. I did see that the playbook rules would not apply. I performed the network capture based on malicious alerts between hosts monitored by SO. There were no suricata alerts for this. But once I ran the replay on the captured traffic I obtained the suricata alerts I expected to see. 
See these below:
![image](https://user-images.githubusercontent.com/15062900/105780552-b5583500-5f3e-11eb-8a89-2c2b2189a8fe.png)

Also, I see that with this replay, I did not get the option to "Show PCAP for this event" when expanding the alerts. Not sure if this means anything. An example is below as well:
![image](https://user-images.githubusercontent.com/15062900/105780692-09631980-5f3f-11eb-9b75-23e36a96286c.png)

**Reply by rammyfzl** [reply]
Hello Folks,

Any thoughts on this peculiar behavior?

Thanks!

**Reply by defensivedepth** [reply]
Is this still an issue?

**Comment by defensivedepth** [comment]
I would also suggest working through our (Free!) Security Onion Essentials Training - https://www.youtube.com/playlist?list=PLljFlTO9rB155aYBjHw2InKkSMLuhWpxH

**Reply by rammyfzl** [reply]
I did do the Essentials training while performing the installation. I may redo it to see if anything was missed. In the meantime are there any further courses that can be suggested.
Thanks
